sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -495,12 +495,22 @@ static Query toQuery(Reference reference, Object value, Context context) {
             }
         }
 
+        class Ignore3vlQuery implements FunctionToQuery {
+
+            @Nullable
+            @Override
+            public Query apply(Function input, Context context) throws IOException {
+                List<Symbol> args = input.arguments();
+                assert args.size() == 1 : "ignore3vl expects exactly 1 argument, got: " + args.size();
```

Comment:
is this needed? afaik this is already checked while resolving the function

Reply:
yeah it was another "safety net". I can remote it. thx!

Reply:
Actually not, sorry, I thought of that before and I retested it. If it's in WHERE clause it goes directly to the LuceneQueryBuilder.